src/libs/ajax/leonardo/models/disk-models.ts

```diff
@@ -61,12 +63,24 @@ export interface ListDiskItem {
 export type PersistentDisk = ListDiskItem | GetDiskItem;
 export type AppDataDisk = PersistentDisk;
 
+// TODO: Rename
 export interface PdType {
+  // TODO: GoogleDiskType is not a label, it is a value
   label: GoogleDiskType;
   displayName: string;
   regionToPricesName: string;
+  // cloudProvider: 'GCP';
 }
 
+export interface AzurePdType {
+  diskType: AzureDiskType;
+  displayName: string;
+  // TODO: Pricing skuLetter: 'S'; Enable SSD types | 'E';
+  // cloudProvider: 'AZURE';
+}
+
+export type SharedPdType = PdType | AzurePdType;
```

Comment:
i'd prefer to call this `PdType` and the current one be renamed to `GooglePdType`